sofiadinizms/HackGrrrl
# Feat/initial configs

## What I did
1. Created the search view
2. Made the search view responsively
3. Created the Price comparison view (not finished yet)

## Hot to test
1. Run `yarn` on the client folder
2. Run `yarn start`
3. Compare and check if its similar to the mockup
4. PS: the price comparison view is not finished yet. I only opened this PR with it still in progress so you can use the basic configs I made to make the next pages, so don't mind this one too much, I'll finish it in the morning
